app/views/incidents/show.pdf.prawn

```diff
@@ -1,6 +1,5 @@
 # PDF is 740 x 560
 
-PrawnRailsForms.default_text_field_options[:style] = :bold
```

Comment:
I removed this functionality and just added `{ style: :bold }` to `PrawnRailsForms::DocumentExtensions#text_field_options`